Formula/overcommit.rb

```diff
@@ -0,0 +1,31 @@
+class Overcommit < Formula
+  desc "Utility to install, configure, and extend Git hooks"
+  homepage "https://github.com/sds/overcommit"
+  url "https://github.com/sds/overcommit/archive/v0.49.1.tar.gz"
+  sha256 "6bf60311211efdd036abe14d62cd314226f99d6c0676810e79df555a72bc9ac3"
+
+  depends_on "ruby"
+
+  resource "childprocess" do
+    url "https://rubygems.org/downloads/childprocess-1.0.1.gem"
+    sha256 "7fe65b8d220967e8b47d8027274db40667641829f12ce9c6145022949c564fb4"
+  end
+
+  resource "iniparse" do
+    url "https://rubygems.org/downloads/iniparse-1.4.4.gem"
+    sha256 "9abf3c9daac0ccaad0829b967e1e98b47ffddba9014295b02fd03d34909971f5"
+  end
+
+  def install
+    ENV["GEM_HOME"] = libexec
+    resources.each do |r|
+      r.fetch
+      system "gem", "install", r.cached_download, "--ignore-dependencies",
+             "--no-document", "--install-dir", libexec
+    end
+    system "gem", "build", "overcommit.gemspec"
+    system "gem", "install", "--ignore-dependencies", "overcommit-#{version}.gem"
```

Comment:
Can `bundle` not be used instead?

Reply:
I copied this one from other formulas.

Reply:
Please try to use `bundle install --standalone` instead. `grep` other formulae to see the usage of that.

Reply:
There are no formulas with `bundle install --standalone` or even `bundler install`.

Reply:
- https://github.com/Homebrew/homebrew-core/blob/1e75c5d171810673203853789c7f87df8b2102ac/Formula/swiftgen.rb#L22-L27
- https://github.com/Homebrew/homebrew-core/blob/1e75c5d171810673203853789c7f87df8b2102ac/Formula/wpscan.rb#L28-L33

Reply:
Correction, the swiftgen formula uses bundler but it doesn't use standalone.

Reply:
Yeh but most of the formulas don't seem to use it and use `gem install` instead like I did.
If we want to change that, that's fine.
We just need to be aware of that fact.